server/requirements.txt

```diff
@@ -0,0 +1,173 @@
+abstract_singleton==1.0.1
```

Comment:
do we need all these dependencies ?

Reply:
All these dependencies are part of flask. 

`pip install Flask`

Requirements.txt file was generated when I ran 
`pip freeze > requirements.txt`
after installing Flask.

So, in short yes those dependencies are required. 
Users can either install it via 
`pip install -r requirements.txt`
or 
`pip install Flask` 
but to ensure correct version has been installed, 
`pip install -r requirements.txt` might be the best choice. 
Thanks.

Reply:
Hi!

Sorry, I had missed the activation of venv, which brought in these many dependencies, I have fixed it now, and pushed the updated requirements.txt file.

Thanks again for reminding.